### PR TITLE
[Doc] kerberos debug instructions

### PR DESCRIPTION
**Pull request by kaisecheng**
add instructions to enable kerberos debug log

Fixes: https://github.com/elastic/logstash/issues/15802

### CODE REVIEW

File: docs/static/troubleshoot/ts-kafka.asciidoc [hunk 1]
```diff
@@ -163,4 +163,17 @@ https://github.com/logstash-plugins/logstash-input-kafka/issues/210
 * If (and only if) upgrading is not possible, set `consumer_threads` to `1`.
 
 
+[discrete] 
+[[ts-kafka-kerberos-debug]]
+===== Debug Kerberos SASL
+
+To troubleshoot authentication failure in Kafka client, add `-Dsun.security.krb5.debug=true` to `jvm.options` and the following lines to `log4j2.properties`
```
Comment (jsvd) [inline docs/static/troubleshoot/ts-kafka.asciidoc:170]:
```suggestion
To troubleshoot authentication failures in Kafka client, add `-Dsun.security.krb5.debug=true` to `jvm.options` and the following lines to `log4j2.properties`:
```

File: docs/static/troubleshoot/ts-kafka.asciidoc [hunk 2]
```diff
@@ -163,4 +163,17 @@ https://github.com/logstash-plugins/logstash-input-kafka/issues/210
...
+
+```
+logger.kafkainput.name = logstash.inputs.kafka
+logger.kafkainput.level = debug
```
Comment (jsvd) [inline docs/static/troubleshoot/ts-kafka.asciidoc:174]:
```suggestion
logger.kafkainput.name = logstash.inputs.kafka
logger.kafkainput.level = debug
logger.kafkaoutput.name = logstash.outputs.kafka
logger.kafkaoutput.level = debug
```

File: docs/static/troubleshoot/ts-kafka.asciidoc [hunk 3]
```diff
@@ -163,4 +163,17 @@ https://github.com/logstash-plugins/logstash-input-kafka/issues/210
...
+logger.kafka.name = org.apache.kafka
+logger.kafka.level = debug
+```
 
+NOTE: Kerberos debug log is not being output through log4j but is directed to the console.
```
Comment (jsvd) [inline docs/static/troubleshoot/ts-kafka.asciidoc:179]:
```suggestion
NOTE: Logging entries for Kerberos are NOT sent through Log4j but go directly to the console.
```

File: docs/static/troubleshoot/ts-kafka.asciidoc [hunk 4]
```diff
@@ -163,4 +163,19 @@ https://github.com/logstash-plugins/logstash-input-kafka/issues/210
...
+```
+logger.kafkainput.name = logstash.inputs.kafka
+logger.kafkainput.level = debug
+logger.kafkaoutput.name = logstash.outputs.kafka
+logger.kafkaoutput.level = debug
+logger.kafka.name = org.apache.kafka
+logger.kafka.level = debug
+```
 
+NOTE: Logging entries for Kerberos are NOT sent through Log4j but go directly to the console.
```
Comment (karenzone) [inline docs/static/troubleshoot/ts-kafka.asciidoc:181]:
```suggestion
[discrete] 
[[ts-kafka-kerberos-debug]]
===== Debug Kerberos SASL

You can set up your machine to help you troubleshoot authentication failures in the Kafka client.

* In `config/jvm.options`, add:
+
[source,txt]
-----
`-Dsun.security.krb5.debug=true` to `jvm.options` 
-----

* In `config/log4j2.properties`, add: 
+
[source,txt]
-----
logger.kafkainput.name = logstash.inputs.kafka
logger.kafkainput.level = debug
logger.kafkaoutput.name = logstash.outputs.kafka
logger.kafkaoutput.level = debug
logger.kafka.name = org.apache.kafka
logger.kafka.level = debug
-----

NOTE: Logging entries for Kerberos are NOT sent through Log4j but go directly to the console.
```

Comment (karenzone) [reply docs/static/troubleshoot/ts-kafka.asciidoc:181]: Proposing a formatting tweak to make sure that the inline `jvm.options` setting doesn't get overlooked, and to make both settings more eye-catching.

Comment (karenzone) [reply docs/static/troubleshoot/ts-kafka.asciidoc:181]: Can we give users more info on next steps or what to look for? Right now, we're helping them set up debugging and then dropping them.

**Review by karenzone — COMMENTED** [review]
Left some comments inline for your consideration--one wrt formatting and one wrt next steps. Please let me know if you'd like to discuss or iterate.

**Comment by karenzone** [comment]
@kaise, thanks for adding this!  I added myself as an assignee fso that we can track this effort in `platform-docs`.

File: docs/static/troubleshoot/ts-kafka.asciidoc (hunk 4 above)
Comment (kaisecheng) [reply docs/static/troubleshoot/ts-kafka.asciidoc]: We mainly want to write down how to enable debug log to check kerberos issues as it is tricky that the log does not send to log4j log. The next step is that user needs to get the hints from debug log. Do you think we need to add a line to remind them to check the log?

**Comment by kaisecheng** [comment]
@karenzone Thanks for the review. I have accepted the suggestion and made a small change.

File: docs/static/troubleshoot/ts-kafka.asciidoc [hunk 5]
```diff
@@ -173,7 +173,7 @@ You can set up your machine to help you troubleshoot authentication failures in
 +
 [source,txt]
 -----
-`-Dsun.security.krb5.debug=true` to `jvm.options` 
+-Dsun.security.krb5.debug=true
```
Comment (karenzone) [inline docs/static/troubleshoot/ts-kafka.asciidoc:176]: Thanks for catching and fixing this, @kaisecheng  :-D

**Review by karenzone — APPROVED** [review]
I left one more comment about aligning the section heading with content in the topic. Please consider it. Either way, LGTM. Thanks for adding this info!

File: docs/static/troubleshoot/ts-kafka.asciidoc [hunk 6]
```diff
@@ -163,4 +163,29 @@ https://github.com/logstash-plugins/logstash-input-kafka/issues/210
 * If (and only if) upgrading is not possible, set `consumer_threads` to `1`.
 
 
+[discrete] 
+[[ts-kafka-kerberos-debug]]
+===== Debug Kerberos SASL
```
Comment (kaisecheng) [inline docs/static/troubleshoot/ts-kafka.asciidoc:168]:
```suggestion
===== Setting up debugging for Kerberos SASL
```

**Pull request merged**